### DISCUSSION TITLE
Raspbian os Ha docker mood audio

### DISCUSSION THREAD

**Discussion by pakoscosta** [indomus/forum#2579] [GENERALE - Discussioni generiche]
Buongiorno oggi sono riuscito ,grazie alla guida di indomus, ad installare home assistant in docker .
Visto che il mio raspy ha una scheda audio hifiberry dac vorrei installare il software moode audio e non so dove iniziare.
Ho cercato anche su docker qualche immagine da installare ma non trovo niente. 
Se questa soluzione in docker non è fattibile per i 2 software ci sono altre vie per i stallare questi due sullo stesso sistema? Grazie

**Comment by KC-inDomus** [comment]
è un po' offtopic, cmq devi trovare l'immagine corretta, sperimentare e capire bene come far funzionare il tutto. in primis accertati che la scheda venga vista correttamente dal sistema operativo.
